basket/news/backends/braze.py

```diff
@@ -326,13 +327,14 @@ def delete(self, email):
         @return: deleted user data if successful
         @raises: BrazeUserNotFoundByEmailError
         """
-        data = self.get(email=email)
-        if not data:
+        data = self.interface.export_users(email=email, fields_to_export=["external_id"])
+        if not data["users"]:
             raise BrazeUserNotFoundByEmailError
 
+        email_id = data["users"][0].get("external_id")
         self.interface.delete_user(email)
-        # return in list to match CTMS.delete
-        return [data]
+        # return in list of email_id to match CTMS.delete
```

Comment:
Can you add a TODO here for also returning `fxa_id` once we have the aliases created? It's gets used here: https://github.com/silverorange/basket/blob/development/basket/admin.py#L214